app/views/content/train-to-be-a-teacher/if-you-dont-have-a-degree.md

```diff
@@ -46,6 +46,23 @@ Qualifications vary depending on the course. For example, you could get QTS with
 
 $international$
 
+## Bursaries for undergraduate degrees
+
+You may be eligible for a [bursary of $bursaries_undergraduate$](https://www.gov.uk/government/publications/funding-initial-teacher-training-itt/funding-initial-teacher-training-itt-academic-year-2024-to-2025#undergraduate-bursary) if you do a secondary maths or physics course that leads to [qualified teacher status (QTS)](/train-to-be-a-teacher/what-is-qts).
+
+You may also be eligible if you do an opt-in QTS course in secondary computing, languages, mathematics or physics. This is where you start a standard undergraduate degree but then opt-in to initial teacher training to get QTS.
+
+The amount you receive depends on the year that you start your QTS course. You do not get a bursary in every year of an undergraduate course. Depending on the type of course you do, you may get the bursary in your final year or final two years.
+
+#### Undergraduate bursary if you're former military personnel
```

Comment:
```suggestion
### Undergraduate bursary if you're former military personnel
```
Currently skipping a heading level - needs to be an h3 not an h4

![image](https://github.com/user-attachments/assets/5349371a-439f-47f1-895d-86b4d07e072f)

Reply:
updated